doc/model/discretization.rst

```diff
@@ -0,0 +1,282 @@
+=======================
+ Navier-Stokes with DG
+=======================
+
+.. _disc-strat:
+
+Discretization Strategy
+=======================
+
+How to discretize the conservation equations with DG, including how to handle the required
+fluxes, particularly in the viscous setting, is a current topic of research and internal
+discussion.  The following references are useful:
+
+* "The DG Book:" Nodal Discontinuous Galerkin Methods, [Hesthaven_2008]_
+* The BR1 method for discretization of Navier-Stokes, [Bassi_1997]_
+* NS with reactions, [Ihme_2014]_, and [Cook_2009]_
+* The BR2 method, [Bassi_2000]_
+* [Ayuso_2009]_
+
+*MIRGE-Com* currently employs a strategy akin to the BR1 algorithm outlined in
+[Bassi_1997]_, but with thermal terms and chemical reaction sources as outlined in
+[Ihme_2014]_ and [Cook_2009]_.
+
+Navier-Stokes BR1
+=================
```

Comment:
```suggestion
Nodal DG for Navier-Stokes
==========================
```
The reason I'm suggesting this is that this content in this section isn't actually BR1-specific!